cosmwasm/contracts/crosschain-registry/src/exports.rs

```diff
@@ -171,8 +180,39 @@ impl<'a> Registries<'a> {
         Ok(receiver)
     }
 
+    /// Get the bech32 prefix for the given chain
+    /// Example: get_bech32_prefix("osmosis") -> "osmo"
+    pub fn get_bech32_prefix(&self, chain: &str) -> Result<String, RegistryError> {
+        self.deps
+            .api
+            .debug(&format!("Getting prefix for chain: {chain}"));
+        let prefix: String = self
+            .deps
+            .querier
+            .query_wasm_smart(
+                &self.registry_contract,
+                &QueryMsg::GetBech32PrefixFromChainName {
+                    chain_name: chain.to_string(),
+                },
+            )
+            .map_err(|e| {
+                self.deps.api.debug(&format!("Got error: {e}"));
+                RegistryError::Bech32PrefixDoesNotExist {
+                    chain: chain.into(),
+                }
+            })?;
+        if prefix.is_empty() {
+            return Err(RegistryError::Bech32PrefixDoesNotExist {
+                chain: chain.into(),
+            });
+        }
```

Comment:
Would this even be possible to store a map from a value to nothing? Or is this just a defense in depth?

Reply:
right, I'm not sure if it's possible, haven't tested it, but an empty string is still a string, so probably?